homeassistant/components/transmission/sensor.py

```diff
@@ -106,18 +106,19 @@ class TransmissionSpeedSensor(TransmissionSensor):
     """Representation of a Transmission speed sensor."""
 
     _attr_device_class = SensorDeviceClass.DATA_RATE
-    _attr_native_unit_of_measurement = UnitOfDataRate.MEGABYTES_PER_SECOND
+    _attr_native_unit_of_measurement = UnitOfDataRate.MEBIBYTES_PER_SECOND
```

Comment:
Should we set a suggested unit of measurement?
As in, Mebibytes is not what an average user would expect to see?

Reply:
MiB is probably fine as a default.
I have a not great monthly data transfer cap, so I have up&down throttled to 50Kib/s each unless I'm actively downloading something important, or have a significant margin to my cap near the end of the month.  Anyone with more than a 10MBit connection will see speeds > 1MiB/s when actively transferring.

Reply:
> MiB is probably fine as a default.

The point is, the average user will not understand the difference between Mebibytes and megabytes. Therefore, suggesting to set the suggested unit to Megabytes in the Home Assistant UI.

Reply:
I think Frenck's suggestion is good 👍